### ISSUE TITLE
Implement keyboard navigation and accessibility for Nyquist effects

### ISSUE THREAD

**Issue by ianalarina** [audacity/audacity#10932]
> Using keyboard arrow keys does not work. 
> 
> https://github.com/user-attachments/assets/f124f622-4f5b-4b29-bafa-1c719d2e6051 

 _Originally posted by @chinakov in [#10486](https://github.com/audacity/audacity/issues/10486#issuecomment-4450423429)_

**Comment by dozzzzer** [comment]
Tested
